api/src/main/java/com/gentics/mesh/util/PasswordUtil.java

```diff
@@ -0,0 +1,42 @@
+package com.gentics.mesh.util;
+
+import java.util.Random;
+
+/**
+ * Utility class which provides methods to handle passwords.
+ */
+public final class PasswordUtil {
+
+	private static final String ALPHA_NUMERIC_READABLE_CHAR = "ABCDEFGHJKLMNPQRTUVWXYZ"
+		+ "2346789"
+		+ "abcdefghjkmnpqrtuvxyz";
+
+	private static final int DEFAULT_PW_LEN = 6;
```

Comment:
Why 6? For a limited set of characters, more then 10 should be the default length.